worker/backend/integration/run

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+readonly CONTAINER_NAME="containerd-integration"
+
+main() {
+  case $1 in
+  containerd)
+    start_containerd
+    ;;
+  test)
+    docker exec $CONTAINER_NAME go test -v .
+    ;;
+  *)
+    echo "Usage: $0 (containerd|test)"
+    ;;
+  esac
+}
+
+start_containerd() {
+  docker run --privileged -it \
+    --name $CONTAINER_NAME \
+    --volume /scratch \
+    --volume gopath:/gopath \
+    --volume $(realpath ../../..):/src \
+    --workdir /src/worker/backend/integration \
+    concourse/unit \
+    /bin/bash -c '
+PATH=/usr/local/concourse/bin \
+	containerd --config <(echo "state = \"/scratch\"")
```

Comment:
```suggestion
	/usr/bin/containerd --config <(echo "state = \"/scratch\"")
```
I think when you override the `PATH` env var, in my case I got
```
/bin/bash: line 1: containerd: command not found
```

Reply:
this is pretty much what I referred to in https://github.com/concourse/concourse/pull/4988#issuecomment-572697492 - the `concourse/unit` that you used did not yet have concourse's `containerd` binaries (which live under `/usr/local/concourse/bin`), only the `containerd` that comes from the `docker` installation (which is of a different version than the one we use).